packages/manager/modules/pci/src/projects/project/baremetal/onboarding/onboarding.component.js

```diff
@@ -7,5 +7,6 @@ export default {
   bindings: {
     addInstance: '<',
     isEnabled: '<',
+    canDisplayNewRegionBanner: '<',
```

Comment:
Seems the feature flipping is here, so why having a date in front ?

Reply:
The feature flipping is here only to manage restriction for EU.
Also, how can i get a end_date when i call the my dedicated ff resource.
As i see from browser network, only the status (true/false) is returned.

Thanks.

Reply:
Inprogress

Reply:
done